site/content/docs/multistep-checks/websockets.md

```diff
@@ -0,0 +1,160 @@
+---
+title: WebSockets
+weight: 20
+menu:
+  resources:
+    parent: "Multistep API checks"
+    identifier: "multistep-checks-websockets"
+cli: true
+---
+
+WebSockets provide a way to establish a persistent, full-duplex communication channel that operates over a single, long-held request, enabling real-time data transfer between a client and a server.
+
+To create a WebSocket check in Checkly, you'll need to write a script that establishes a WebSocket connection, sends messages, and validates responses.
+
+## Set Up the WebSocket Connection
```

Comment:
@checklyalex this is too drawn out, let's not have a chapter for each small bit of the check. Let's instead have two chapters, one for the generic websocket check code, with comments, an one for the example.